docs/release-notes/v2.26/index.md

```diff
@@ -0,0 +1,64 @@
+# November.1 2020 (version 2.26)
+
+---
+
+Welcome to the November.1 2020 release of Gatsby (version 2.26). Key highlights of this release:
```

Comment:
"the November.1 2020 release" reads awkward to me. It's bikeshedding but I think the `.1` part should be after the year or something else entirely.

Reply:
Yup, let's use version numbers for now (at least until we actually switch to monthly cadence)